src/main/java/com/lorepo/icplayer/client/module/addon/AddonPresenter.java

```diff
@@ -653,4 +653,20 @@ public List<ScoreWithMetadata> getScoreWithMetadata() {
 
 		return addonScoreWithMetadata.getScoreWithMetadata();
 	}
+
+	@Override
+	public boolean isActivity() {
+		for (IProperty property: model.getProperties()) {
+			String propertyName = property.getName().toLowerCase();
+			if (propertyName.startsWith("is") && propertyName.endsWith("activity")) {
```

Comment:
Ten warunek nie pokrywa addonu Plot. Tam property nazywa się "Not Activity". 

Jednak ten addon także nie wspiera GSA. Więc jeżeli rozchodzi się o zadanie to jest to wystarczające rozwiązanie. Jednak w przyszłości może powodować błędy dla Plot jeżeli będzie ta metoda używana. 

Proponuję poszerzenie warunku na:
if ((propertyName.startsWith("is") || propertyName.startsWith("not")) && propertyName.endsWith("activity")) {

Reply:
Po uzgodnieniu daję to do poprawy